staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags.go

```diff
@@ -37,11 +37,14 @@ import (
 )
 
 const (
+	flagKubeconfig         = "kubeconfig"
 	flagClusterName        = "cluster"
 	flagAuthInfoName       = "user"
 	flagContext            = "context"
 	flagNamespace          = "namespace"
+	shortFlagNamespace     = "n"
```

Comment:
For example, how do we know that we are covering everything in addition to `n` and `s` and how people in the future will know that new shortcuts should be added in here?